pointcloud/basic_octree.go

```diff
@@ -182,3 +189,158 @@ func (octree *BasicOctree) CollidesWithGeometry(geom spatialmath.Geometry, thres
 	}
 	return false, errors.New("unknown octree node type")
 }
+
+// Pose returns the pose of the octree.
+func (octree *BasicOctree) Pose() spatialmath.Pose {
```

Comment:
See above comment. I dont think asking for an octree's pose is a valid request. Only requesting center and then casing it to a pose with a default orientation

Reply:
When observing an octree from the perspective of something else, it's totally valid for an octree to have a pose.

If we're not observing the octree from the perspective of something else, then it's not valid for it to have a center, either. The center would/should always be (0,0,0) in that case.